src/util.test.ts

```diff
@@ -31,8 +32,18 @@ describe('util', () => {
     nock.cleanAll();
   });
 
-  it('bNToHex', () => {
-    expect(util.BNToHex(new BN('1337'))).toBe('0x539');
+  describe('BNToHex', () => {
```

Comment:
Updated the tests on this function because I wanted to make sure it did what I expected and I found the tests lacking.